flash

```diff
@@ -704,6 +703,11 @@ if [ -f "${boot}/device-init.yaml" ]; then
   fi
 fi
 
+# set fake-hwclock
+if [ -f "${boot}/fake-hwclock.data" ]; then
+  date --utc --rfc-3339=seconds |cut -d+ -f1 > "${boot}/fake-hwclock.data"
```

Comment:
These `date` options don't not work on macOS, but the following works both on Linux and macOS.

```suggestion
  TZ=UTC date '+%Y-%m-%d %H:%M:%S' > "${boot}/fake-hwclock.data"
```